lib/parser.js

```diff
@@ -11,21 +11,24 @@ const YAML = require('yaml');
  * Returns the proper value for the provided modifier
  */
 function applyModifiers(modifier, config) {
-  let type;
+  let value;
   switch (modifier) {
     case `${config.modifiers.array}`:
-      type = '[]';
+      value = '[]';
       break;
     case `${config.modifiers.object}`:
-      type = '{}';
+      value = '{}';
       break;
     case `${config.modifiers.string}`:
-      type = '""';
+      value = '""';
+      break;
+    case `${config.modifiers.check}`:
+      value = 'Check values.yaml';
```

Comment:
You should not replace the default value with `Check values.yaml` when reading it.

That would cause the metadata to also show the default value as `Check values.yaml`.
Instead, you should add a control variable and modify it when writing the README.md